easy_my_coop_initial_import/models/subscription_request.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class SubscriptionRequest(models.Model):
+    _inherit = "subscription.request"
+
+    migrated_cooperator_register_number = fields.Integer(
+        string="Migrated Cooperator Number"
```

Comment:
To follow the module name, maybe "imported_cooperator_register_number" would be more consistent.

Can you also add a `help` attribute to explain it comes from the initial csv import ?